exercises/05.complex-structures/01.problem.nested/app/routes/users+/$username_+/notes.$noteId_.edit.tsx

```diff
@@ -239,7 +239,7 @@ function ImageChooser({
 							)}
 							{existingImage ? (
 								// 🐨 update this to use the conform.input helper on
-								// fields.image.id (make sure it stays hidden though)
+								// fields.id (make sure it stays hidden though)
```

Comment:
you correct this in the video, not sure if you also want to correct in the instruction here?